index.html

```diff
@@ -1913,6 +1919,64 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    [=People=] should be provided relevant information when data is accessed or when permission is
+    requested.
```

Comment:
Might it make sense to  flip this around to be more active? Maybe: "Sites and implementations should provide people with relevant information (about the intended processing) when accessing their data or requesting their permission."